src/Sql/InsertIgnore.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright Copyright (c) 2005-2016 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+
+namespace Zend\Db\Sql;
+
+class InsertIgnore extends Insert
+{
+    /**
+     * @var array Specification array
```

Comment:
Please specify the type of the array

Reply:
it is copied originally from `Insert` class, do you have sample of "type of the array"?

Reply:
`string[]` in this case, or even `array<string, string>` (unsure if the tooling craps out with that, though)